Sources/CorePayments/Networking/AnalyticsService.swift

```diff
@@ -28,7 +29,7 @@ public struct AnalyticsService {
     
     // MARK: - Public Methods
         
-    /// :nodoc: This method is exposed for internal PayPal use only. Do not use. It is not covered by Semantic Versioning and may change or be removed at any time.
+    /// This method is exposed for internal PayPal use only. Do not use. It is not covered by Semantic Versioning and may change or be removed at any time.
```

Comment:
I think we want a `_documentation` annotation below

Reply:
Wow good catch!

Reply:
Ah, you may be right. You can check in Xcode by going to `Product` > `Build Documentation` and verifying this is hidden as expected.

Reply:
Nice. No, I don't see it but is Demo app stuff supposed to be in there?